index.html

```diff
@@ -1451,7 +1451,15 @@ <h2>Core Vocabulary Definitions</h2>
           <code>unsubscribeevent</code>, or both terms within an
           <a href="#dfn-array" class="internalDFN" data-link-type="dfn">Array</a>.</span></p><p class="note">
 		  It is considered to be good practice that each <code>subscribeevent</code> has a corresponding <code>unsubscribeevent</code> unless the protocol
-		  supports implicit unsubscription mechanisms (e.g., heartbeat to detect connection loss).</p></section>
+		  supports implicit unsubscription mechanisms (e.g., heartbeat to detect connection loss).</p>
+          <p>
+            EventAffordances are similar to observable PropertyAffordances in the sense that the Thing itself informs interested Consumers about state changes.
+            However, a main difference of EventAffordances is that not every change of the associated resource needs to trigger an event message to be emitted, e.g. a critical threshold for a numeric value.
+        Furthermore, EventAffordances allow for more complex subscribing and unsubscribing mechanisms via the definition of <code>subscription</code>, <code>dataResponse</code>, and <code>cancellation</code> DataSchema definitions.
```

Comment:
Not exactly where but I would add the fact that if an affordance can be read, it will be a property, probably.